multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperContentUtil.java

```diff
@@ -46,7 +46,11 @@ public class MapperContentUtil {
 
   private static final String ARRAY = "array";
 
-  private static final String BIG_DECIMAL = "bigDecimal";
+  private static final String BIG_DECIMAL = "BigDecimal";
+
+  private static final String LOCAL_DATE = "LocalDate";
+
+  private static final String LOCAL_DATE_TIME = "LocalDateTime";
```

Comment:
yes that's correct but there is no simple ZonedDate
I want to make it conferrable and but then the complexity will grow up at function parameters more :(

Reply:
That is ok. We can stick with LocalDate since time zone don't matter quite much. We have another project were we deal with that and we left dates are LocalDate but date time we use ZonedDateTime.